resources/views/themes/bimbala/about_us.blade.php

```diff
@@ -2,26 +2,35 @@
 
 @section('content')
     {{--Contact--}}
-    <section class="w-full relative">
-        <div class="h-full inset-x-0 bottom-0 absolute"></div>
-        <div class="max-w-7xl px-12 py-12 lg:py-24  mx-auto text-center relative">
+    <section class="max-w-6xl mx-auto relative ">
+        <div class="px-12 pt-12 pb-20">
             <div class="space-y-12 text-justify">
-                <div class="max-w-3xl mb-20 space-y-5 sm:mx-auto sm:space-y-4">
-                    <h3 class="relative text-3xl tracking-tight sm:text-4xl text-black mt-4">Values</h3>
-                    <p class="text-xl text-gray-600">
+                <div class="mb-20 space-y-5 sm:mx-auto sm:space-y-4">
+                    <h3 class="relative text-3xl tracking-tight sm:text-4xl mt-4">Values</h3>
+                    <p>
                         The first principle of our company is to be transparent. Honesty is something we value the most.
-                        We are always aiming to show our customers the things as they are.
+                        We are always aiming to show our customers the things as they are. We maintain open culture and believe that everyone is a contributor and has something to bring to the table.
                     </p>
 
-                    <h3 class="relative text-3xl tracking-tight sm:text-4xl text-black mt-4">Target</h3>
-                    <p class="text-xl text-gray-600">
+                    <h3 class="relative text-3xl tracking-tight sm:text-4xl mt-4">Target</h3>
+                    <p>
                         The product is carried mostly on the shoulders of two university students.
                          Our dream is to succeed in addressing the needs of each business that carries about its customers' feedback.
                          This includes easier work process for managers as well as all desperate employees from a support team.
                     </p>
+                    <h3 class="relative text-3xl tracking-tight sm:text-4xl mt-4">Community</h3>
+                    <p>
+                        Understanding our <a href="/discord">community</a> will be the key to our success. We are always open to feedback and suggestions.
```

Comment:
Add some more styling to the link here. It doesn't stand out from the rest of the text.

Reply:
Added some styling on both of the links so they stand out.